EKF/ekf_helper.cpp

```diff
@@ -982,12 +982,62 @@ void Ekf::get_gyro_bias(float bias[3])
 	memcpy(bias, temp, 3 * sizeof(float));
 }
 
+// get the full covariance matrix
+const matrix::SquareMatrix<float, 24> Ekf::covariances() const
```

Comment:
Can be removed (single line in header).

Reply:
I was actually trying `matrix::Vector<float, 24> Ekf::covariances_diagonal() const { return matrix::Vector<float, 24>{P}; }` by your suggestion but that doesn't work.

Reply:
Matrix or SquareMatrix, not Vector.